app/views/routes_into_teaching/steps/completed.html.erb

```diff
@@ -46,7 +44,10 @@
           link_text: result["cta_text"],
           link_target: result["cta_link"],
         ) %>
-        </div>
+          </div>
+        <% end %>
+      <% else %>
+        <p>There are no teacher training routes that match your current circumstances. To find out more about routes into teaching, <%= link_to "speak to one of our teacher training advisers", "/teacher-training-advisers" %>.</p>
```

Comment:
@Sarah-DfE I think this needs to point to the contact centre instead as not everyone will be eligible for an adviser?